### ISSUE TITLE
replace in stringSanitizer.js

### ISSUE THREAD

**Issue by ChandreshL** [felipe-augusto/linguee#7]
TypeError: Cannot read property 'replace' of undefined
    at Object.removeNonBreakableSpace (c:\path\node_modules\linguee\utils\stringSanitizer.js:4:18)

------
Posible solution

 if(str)
        return str.replace(/\u00A0/g, ' ');
      else
        return str;

**Comment by einenlum** [comment]
Hi @ChandreshL, thank you for your feedback.

This is strange since `str` is set to `''` by default in `function(str = '')`.
Would you have more details and a concrete example where it fails?

Thanks!

**Comment by ChandreshL** [comment]
Hello, @Einenlum,

I installed the library and tested it with the example code that is on the README.md
Only change I made was in the options in translate function as, `{ from: 'ger', to: 'eng' }`

I got version 1.0.0 from npm and it doesn't have `str` set to `''` by default!
But the one of the commit on the github updates it to default empty string.

It is useful to me to create my own word lists, and saves me a lot of time!

Thanks
